### DISCUSSION TITLE
Can't load YouTube channel item into a Group

### DISCUSSION THREAD

**Discussion by Papoulka** [mattwright324/youtube-comment-suite#121] [General]
Running Windows 10 x64 with YCS v1.4.8

I have a new Group with no items.  I'm trying to add this URL:
[https://www.youtube.com/c/JohnBartholomewChess](url)
But it always brings up the error "There was a problem, could not resolve link".

I've seen the same problem with other channels that have a "/c" in the URL, but don't know if that is actually relevant.

In contrast, this similar channel adds instantly:
[https://www.youtube.com/user/PowerPlayChess](url)

Thanks for any help

**Comment by mattwright324** [comment]
Yup, so the issue is that the YouTube API does not officially provide an easy way of resolving `/c/` custom channel URLs. I have the issue in more detail here: https://github.com/mattwright324/youtube-metadata/issues/1

You should be able to follow one of the manual resolving methods (2a or 2b) for custom channels that do not seem to work. The issue is more tailored for that other application (YouTube Metadata) than this one (YCS) but can generally be followed.

**Comment by Papoulka** [comment]
Thanks for the quick response.  I used Method 2a "Inspect Element" which worked fine.  I've added that to the relevant thread on Github.

***@***.***
